### PR TITLE
alpha-0.1.7/277-add camera feed to dashboard

### PR DESCRIPTION
**Pull request by 1Blademaster**
Adds a panel to the dashboard tabs which allows a user to add a camera input and displays the video feed. Used for video feeds, e.g. if an FPV system outputs video to the GCS computer.

### CODE REVIEW

**Comment by NexInfinite** [comment]
I'll code review in a bit/tomorrow, can't actually test it on my pc as I don't have a FPV camera - can you provide screenshots of it working?

File: gcs/src/dashboard.jsx [hunk 1]
```diff
@@ -419,7 +435,7 @@ export default function Dashboard() {
             }}
             className='h-full'
           >
-            <div className='flex flex-col p-2 h-full gap-2'>
+            <div className='flex flex-col p-1 h-full gap-2 overflow-x-hidden overflow-y-auto'>
```
Comment (NexInfinite) [inline gcs/src/dashboard.jsx:438]: overflow hidden could create issues for joe's changes, is this needed? #306

**Review by NexInfinite — CHANGES_REQUESTED** [review]

File: gcs/src/dashboard.jsx (hunk 1 above)
Comment (1Blademaster) [reply gcs/src/dashboard.jsx:438]: It adds scrollbar and stuff, idk what Joe's doing so not sure, we can merge and he can change it afterwards if he needs to?

Comment (NexInfinite) [reply gcs/src/dashboard.jsx:438]: Sure

**Review by NexInfinite — APPROVED** [review]
LGTM

**Pull request merged**